docs/administer/rbac.md

```diff
@@ -0,0 +1,362 @@
+
+            - **namespace**:  `Read` access to the `the-dark-side`  .
+            - **Database engines**: `Read` access to all the database engines
+            - **Database clusters**:  Read and update access for **only** the `banana` database
+            - **Database clusters credentials**: `Read` acccess for **only** the `banana` database
+            - **Backup storages**: Read access to all the backup storages
+            - **Monitoring instances**: Read access to all the monitoring instances
+
+
+    3. **Dev group role definition with read/update/create access to all the namespaces**
+
+        Let's set up a dev group role with **read/update/create** access to **all** the namespaces:
+
+        ```sh
+        p, team-darkside:role, namespaces, read, */*
+        p, team-darkside:role, database-engines, read, */*
+        p, team-darkside:role, database-clusters, read, */*
+        p, team-darkside:role, database-clusters, update, */*
+        p, team-darkside:role, database-clusters-credentials, read, */*
+        p, team-darkside:role, backup-storages, read, */*
+        p, team-darkside:role, monitoring-instances, read, */*
+        ```
+
+        ??? info "Let's dive into decoding this!"
+
+            The `team-darkside` role has the following privileges in **all** the namespaces:
+
+            - **namespace**:  `Read` access in all the namespaces.
+            - **Database engines**: `Read` access to all the database engines
+            - **Database clusters**:  `Read` and update access for all the database clusters
+            - **Database clusters credentials**: `Read` acccess for **all** the databases
+            - **Backup storages**: `Read` access to all the backup storages
+            - **Monitoring instances**: `Read` access to all the monitoring instances
+
+
+## Assigning roles to users
+
+In order for roles to take effect, they need to be assigned to users. The syntax for this is as follows:
+
+```sh
+g, username, rolename
+```
+
+A new user in Percona Everest will initially have **no** permissions. To grant permissions, you must edit your RBAC configuration stored in the `everest-rbac` `ConfigMap` in the `everest-system` namespace:
+
+
+```sh
+kubectl edit configmap-rbac -n everest-system
+```
+
+A text editor will open, and you can edit the `ConfigMap` as follows. You just have to add the new user and assign it the desired role.
+
+
+```sh
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: everest-rbac
+data:
+  policy.csv: |
+    p, adminrole:role, namespaces, read, *
+    p, adminrole:role, database-engines, *, */*
+    p, adminrole:role, database-clusters, *, */*
+    p, adminrole:role, database-cluster-backups, *, */*
+    p, adminrole:role, database-cluster-restores, *, */*
+    p, adminrole:role, database-cluster-credentials, read, */*
+    p, adminrole:role, backup-storages, *, */*
+    p, adminrole:role, monitoring-instances, *, */*
```

Comment:
We have [updated the adminrole](https://github.com/percona/everest/pull/649/files). Please reflect the changes here:
```suggestion
    p, adminrole:role, namespaces, *, *
    p, adminrole:role, database-engines, *, */*
    p, adminrole:role, database-clusters, *, */*
    p, adminrole:role, database-cluster-backups, *, */*
    p, adminrole:role, database-cluster-restores, *, */*
    p, adminrole:role, database-cluster-credentials, *, */*
    p, adminrole:role, backup-storages, *, */*
    p, adminrole:role, monitoring-instances, *, */*
```